modules/packages/LinearAlgebra.chpl

```diff
@@ -419,11 +425,17 @@ proc matMinus(A: [?Adom] ?eltType, B: [?Bdom] eltType) where isDefaultRectangula
 }
 
 /* Subtract matrices, maintaining dimensions */
-proc _array.minus(A: [?Adom]) where isDefaultRectangularArr(A) && isDefaultRectangularArr(this) {
```

Comment:
You modified the dense case - you should modify the sparse case defined in `LinearAlgebra.Sparse` instead

Reply:
this is still flagged because it was previously missing a type query `?eltType` on `A` i think